src/control/client/mocks.go

```diff
@@ -242,14 +242,27 @@ func newMockMgmtCtlClient(
 type mockMgmtSvcClient struct{}
 
 func (m *mockMgmtSvcClient) CreatePool(
-	ctx context.Context, req *pb.CreatePoolReq, o ...grpc.CallOption) (
-	*pb.CreatePoolResp, error) {
+	ctx context.Context,
+	req *pb.CreatePoolReq,
+	o ...grpc.CallOption,
```

Comment:
Is it intentional that these mock methods accept variadic grpc.CallOption parameters but the "real" ones don't? They don't seem to be used by anything.

Reply:
this seems to match the signature https://github.com/daos-stack/daos/pull/666/files/8359a62035c2437f8f51c86833f5506b39ef77b2#diff-b69822f767434d1ebdc84f04c4e57d77R339 that we are implementing, doesn't it?